src/Tribe/Assets.php

```diff
@@ -167,12 +167,19 @@ public static function maybe_get_min_file( $url ) {
 
 		// Check for all Urls added to the array
 		foreach ( $urls as $key => $url ) {
-			// Turn URL into file Path
-			$file = str_replace( WP_CONTENT_URL, WP_CONTENT_DIR, $url );
+			//set path to file for Windows
+			$file = $url;
 
-			// Make it compatible with Windows and other OS
-			$file = str_replace( '/', DIRECTORY_SEPARATOR, $file );
+			//Detect if $url is actually a file path
+			if (  false !== strpos($url, wp_normalize_path( WP_CONTENT_DIR )) ) {
+				// Turn file Path to URL in Windows
+				$url = str_replace( wp_normalize_path( WP_CONTENT_DIR ), WP_CONTENT_URL, $url );
+			} else {
+				// Turn URL into file Path
+				$file = str_replace( WP_CONTENT_URL, wp_normalize_path( WP_CONTENT_DIR ), $url );
+			}
```

Comment:
This check was added as Windows the $url would be: C:/xampp/htdocs/events/wp-content/plugins/the-events-calendar/src/resources/js/aggregator-fields.js